dist/index.html

```diff
@@ -119,39 +119,39 @@ <h2 class="fanFavoriteTrips">Fan Favorite Trips</h2>
-                  <option date-index="1" value="New New York, New York">New York, New York</option>
-                  <option date-index="1" value="London, England">London, England</option>
-                  <option date-index="1" value="San Francisco, California">San Francisco, California</option>
-                  <option date-index="1" value="San Jaun, Puerto Rico">San Jaun, Puerto Rico</option>
-                  <option date-index="1" value="Willemstad, Curaçao">Willemstad, Curaçao</option>
+                  <option value="Lima, Peru">Lima, Peru</option>
+                  <option value="Stockholm, Sweden">Stockholm, Sweden</option>
+                  <option value="Sydney, Austrailia">Sydney, Austrailia</option>
+                  <option value="Cartagena, Colombia">Cartagena, Colombia</option>
+                  <option value="Madrid, Spain">Madrid, Spain</option>
+                  <option value="Jakarta, Indonesia">Jakarta, Indonesia</option>
+                  <option value="Paris, France">Paris, France</option>
+                  <option value="Tokyo, Japan">Tokyo, Japan</option>
+                  <option value="Amsterdam, Netherlands">Amsterdam, Netherlands</option>
+                  <option  value="Toronto, Canada">Toronto, Canada</option>
+                  <option  value="Mikonos, Greece">Mikonos, Greece</option>
+                  <option  value="Wellington, New Zealand">Wellington, New Zealand</option>
+                  <option  value="St. Petersburg, Russia">St. Petersburg, Russia</option>
+                  <option  value="Marrakesh, Morocco">Marrakesh, Morocco</option>
+                  <option  value="Manila, Philippines">Manila, Philippines</option>
+                  <option  value="Bangkok, Thailand">Bangkok, Thailand</option>
+                  <option  value="Jaipur, India">Jaipur, India</option>
+                  <option  value="Cape Town, South Africa">Cape Town, South Africa</option>
+                  <option  value="Quito, Ecuador">Quito, Ecuador</option>
+                  <option  value="Miami, Florida">Miami, Florida</option>
+                  <option value="Tulum, Mexico">Tulum, Mexico</option>
+                  <option value="Rome, Italy">Rome, Italy</option>
+                  <option value="Copenhagen, Denmark">Copenhagen, Denmark</option>
+                  <option value="Vilnius, Lithuania">Vilnius, Lithuania</option>
+                  <option value="New New York, New York">New York, New York</option>
+                  <option value="London, England">London, England</option>
+                  <option value="San Francisco, California">San Francisco, California</option>
+                  <option value="San Jaun, Puerto Rico">San Jaun, Puerto Rico</option>
+                  <option value="Willemstad, Curaçao">Willemstad, Curaçao</option>
                   <option value="Antananarivo, Madagascar">Antananarivo, Madagascar</option>
```

Comment:
wow, you did a lot of work! potential refactoring option: maybe using a forEach and innerHTML to populate the whole select options?